DEVELOPER.md

```diff
@@ -102,18 +102,18 @@ need to:
    account.  See [Fork a repo](https://help.github.com/articles/fork-a-repo) for detailed instructions.
 2. Commit any changes to your fork.
 3. Send a [pull request](https://help.github.com/articles/creating-a-pull-request) to the Streamline GitHub repository
-   that you forked in step 1.  If your pull request is related to an existing IoTaS JIRA ticket -- for instance, because
-   you reported a bug report via JIRA earlier -- then prefix the title of your pull request with the corresponding JIRA
-   ticket number (e.g. `IOT-123: ...`).
+   that you forked in step 1.  If your pull request is related to an existing Github issue  -- for instance, because
+   you reported a bug report via Github issue earlier -- then prefix the title of your pull request with the corresponding 
+   Github issue number (e.g. `ISSUE-123: ...`).
 
 You may want to read [Syncing a fork](https://help.github.com/articles/syncing-a-fork) for instructions on how to keep
-your fork up to date with the latest changes of the upstream  `Streams` repository.
+your fork up to date with the latest changes of the upstream  `Streamline` repository.
 
 ### Git Commit Messages Format
```

Comment:
Should this be changed to PR message format since the script will put the PR message as the commit message ?

Reply:
Sure, but I also think we should apply template of pull request body. For now I will just change this to PR format.

Reply:
@arunmahadevan Also addressed.